### PR TITLE
markdownify

### PR DESCRIPTION
**Pull request by omercnet**
## Status
WIP

## Related Issues
fixes: https://github.com/demisto/etc/issues/20559
depends on: https://github.com/demisto/dockerfiles/pull/1049

## Description
An automation to convert html to markdown

## Screenshots
image

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/tomd/Scripts/markdownify/markdownify.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/tomd/Scripts/markdownify/CHANGELOG.md)

### CODE REVIEW

File: Scripts/tomd/tomd.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import demistomock as demisto
+from CommonServerPython import *
+import tomd
```
Comment (amshamah419) [inline Scripts/tomd/tomd.py]:
```suggestion
from markdownify import markdownify as md
```

File: Scripts/tomd/tomd.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+return_outputs(tomd.convert(demisto.getArg('html')))
```
Comment (amshamah419) [inline Scripts/tomd/tomd.py]:
```suggestion
return_outputs(md(demisto.getArg('html')))
```

File: Scripts/tomd/tomd.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+tags: []
+enabled: true
+args:
+- name: html
+  required: true
+  default: true
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/tomd
```
Comment (amshamah419) [inline Scripts/tomd/tomd.yml]: https://github.com/matthewwithanm/python-markdownify is what I'm using in the AWS CodeGen tool.

**Comment by omercnet** [comment]
converted to use markdownify thanks @amshamah419 !

**Comment by Itay4** [comment]
@omercnet whats our status here?

**Comment by omercnet** [comment]
@Itay4 it's awaiting review

**Comment by Itay4** [comment]
@amshamah419 can you review?

File: Scripts/markdownify/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## [Unreleased]
+
+  - Created an automation to convert html to markdown
```
Comment (amshamah419) [inline Scripts/markdownify/CHANGELOG.md]:
```suggestion
## [Unreleased]
  - Created an automation to convert html to markdown
```

File: Scripts/markdownify/markdownify.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+enabled: true
+args:
+- name: html
+  required: true
+  default: true
+  description: "HTML to convert to Markdown"
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/btfl-soup:1.0.1.4581
```
Comment (amshamah419) [inline Scripts/markdownify/markdownify.yml]:
```suggestion
dockerimage: demisto/btfl-soup:1.0.1.6117
```

File: Scripts/markdownify/markdownify.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+enabled: true
+args:
+- name: html
+  required: true
+  default: true
+  description: "HTML to convert to Markdown"
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/btfl-soup:1.0.1.6117
```
Comment (amshamah419) [inline Scripts/markdownify/markdownify.yml]:
```suggestion
dockerimage: demisto/btfl-soup:1.0.1.6233
```

File: TestPlaybooks/playbook-Test_Markdownify.yml [hunk 7]
```diff
@@ -0,0 +1,139 @@
+id: Test_Markdownify
+version: 4
```
Comment (bakatzir) [inline TestPlaybooks/playbook-Test_Markdownify.yml]:
```suggestion
version: -1
```

**Comment by kirbles19** [comment]
@omercnet Done.

**Comment by Itay4** [comment]
Closing this due to changes done in `master` branch.
Please reopen in a new branch.

**Pull request closed without merging**